### PR TITLE
Edit project get api

### PR DESCRIPTION
**Pull request by Roja1912**
@HarshadKavathiya Please do review for change in Project GET API

### CODE REVIEW

File: application/api/project.py [hunk 1]
```diff
@@ -133,16 +134,12 @@ def get(self, session):
...
-        list_of_active_project = Project.query.filter_by(
-            org_id=get_project_data['org_id'], is_deleted=False).all()
-        if not list_of_active_project:
-            return api_response(False,
-                                APIMessages.NO_RESOURCE.format('Project'),
-                                STATUS_UNAUTHORIZED)
-        check_permission(user_object=session.user,
-                         list_of_permissions=PROJECT_GET,
-                         org_id=get_project_data["org_id"])
+        user_obj = User.query.filter(
```
Comment (HarshadKavathiya) [inline application/api/project.py]: replce 
user_obj = User.query.filter(
            User.user_id == session.user_id).first()
with 
user_obj = session.user

File: application/api/project.py [hunk 2]
```diff
@@ -152,17 +149,54 @@ def get(self, session):
...
+                 'is_org_user': True if user_roles else False,
+                 'project_details': project_details_list})
+            return api_response(
+                True, APIMessages.SUCCESS, STATUS_OK,
+                {"projects_under_organization": projects_to_return})
+        else:
+            project_obj = UserProjectRole.query.filter(
+                UserProjectRole.user_id == session.user_id,
+                UserProjectRole.org_id == get_project_data['org_id']).all()
+            if project_obj == []:
```
Comment (HarshadKavathiya) [inline application/api/project.py]: replace 

`if project_obj == []:`

with 

`if project_obj`:

File: application/api/project.py [hunk 3]
```diff
@@ -152,17 +149,54 @@ def get(self, session):
...
+                return api_response(False,
+                                    APIMessages.NO_RESOURCE.format('Project'),
+                                    STATUS_UNAUTHORIZED)
+            active_project = []
+            for each_project in project_obj:
```
Comment (HarshadKavathiya) [inline application/api/project.py]: Please optimize this logic.

File: application/api/project.py (hunk 1 above)
Comment (Roja1912) [reply application/api/project.py]: ok

File: application/api/project.py (hunk 2 above)
Comment (Roja1912) [reply application/api/project.py]: done

File: application/api/project.py (hunk 3 above)
Comment (Roja1912) [reply application/api/project.py]: ok,done

**Pull request merged**